src/app/templates/app/submit_new_license.html

```diff
@@ -352,9 +352,11 @@
               data: form,
               success: function (data) {
                 var githubCode = data.statusCode;
+                var githubIssueId = data.issueId;
                 if(githubCode == '201'){
                   var successMessage = "The license request has been successfully submitted.";
                   $("#messages").html('<div class="alert alert-success alert-dismissable fade in"><a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a><strong>Success! </strong>'+ successMessage +'</div>');
+                  window.location.href = "https://github.com/spdx/licnse-list-xml/issues/" + githubIssueId;
```

Comment:
I don't think directly transferring users to a new website is ideal. We should either redirect to our own license ID page which should list down the github url (i think it doesn't now) or open this github url in a new tab

Reply:
Yes @rtgdk , I had also suggested the same idea like once there is a successful license submission the user should get redirected to 'https://tools.spdx.org/app/license_requests/{id}' wherein they can see more about his license instead of just again redirecting them to `submit new license`. [Discussion](https://github.com/spdx/license-list-XML/issues/1889) but @jlovejoy suggested that currently the working is like it gets redirected to the github page which was not working when I tried it.

Reply:
Yeah opening github link looks fine - both technically and legally. But this should get opened in a new tab. The user should still have a tab to navigate spdx tools.

Reply:
Ok, I will update the PR according so that whenever the new license get's submitted it gets opened in a new tab